assets/src/hooks/useMinischedule.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react"
+import { fetchMinischeduleBlock, fetchMinischeduleRun } from "../api"
+import { Block, Run } from "../minischedule"
+import { TripId } from "../schedule"
+
+export const useMinischeduleRun = (tripId: TripId): Run | null => {
+  const [run, setRun] = useState<Run | null>(null)
+  useEffect(() => {
+    fetchMinischeduleRun(tripId).then(setRun)
```

Comment:
These API calls can return `null` as a legitimate response, right? Do we want to set the state to be `undefined` initially to distinguish between loading and empty results?